File: LargestNDigitSquare.py
    from math import sqrt;
     
    def formTheNumber(n):
    	string = '';
    	for i in range(0, n, 1):
    		string += '9';
    		
    	return long(string);
     
    root = long(sqrt(formTheNumber(5)));
    print(root * root);

File: Java/LargestNDigitSquare.java
public class LargestNDigitSquare {

	public static void main(String[] args) {
		long root = (long)Math.sqrt(formTheNumber(5));
		System.out.println(root * root);
	}
	private static long formTheNumber(int n)
	{
		StringBuilder stringBuilder = new StringBuilder();
		
		for(int i = 0; i < n; ++ i)
			stringBuilder.append(9);
		
		return Long.parseLong(stringBuilder.toString());
	}
}
